YPImagePickerExample/YPImagePickerExample.xcodeproj/project.pbxproj

```diff
@@ -431,7 +434,7 @@
 			buildSettings = {
 				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
```

Comment:
Could you remove all these below from the PR?  `DEVELOPMENT_TEAM` & `PRODUCT_BUNDLE_IDENTIFIER`?

Reply:
I'm sorry. I'm going to change it now.